x/launchpad/keeper/genesis.go

```diff
@@ -71,3 +78,26 @@ func (k Keeper) exportUserPositions(moduleStore prefix.Store) ([]types.UserPosit
 
 	return res, nil
 }
+
+func (k Keeper) importSales(moduleStore prefix.Store, sales []types.Sale) error {
+	for _, sale := range sales {
+		idBZ := make([]byte, 8)
+		binary.BigEndian.PutUint64(idBZ, sale.Id)
+		// TODO: do we need any validation here ?
+		k.saveSale(moduleStore, idBZ, &sale)
+	}
+	return nil
+}
+
+func (k Keeper) importUserPositions(moduleStore prefix.Store, userPositionKv []types.UserPositionKV) error {
+    for _, up := range userPositionKv {
+		idBZ := make([]byte, 8)
+		binary.BigEndian.PutUint64(idBZ, up.SaleId)
+		address, err := sdk.AccAddressFromBech32(up.AccAddress)
+		if err != nil {
+			return err
```

Comment:
let's wrap an error and add more information (index of in the `userPositinKv` list